instrumentation.ts

```diff
@@ -0,0 +1,28 @@
+import * as Sentry from "@sentry/nextjs"
+
+export async function register() {
+  const dns = process.env.NEXT_PUBLIC_SENTRY_DSN
```

Comment:
```suggestion
  const dsn = process.env.NEXT_PUBLIC_SENTRY_DSN
```

I think renaming this variable would be a bit more clear since we are getting NEXT_PUBLIC_SENTRY_DSN and passing as dsn props in commonSentryOptions

Reply:
oops, thanks! a typo confused by "domain name server" xD